release-notes/opensearch-alerting.release-notes-1.1.0.0.md

```diff
@@ -26,6 +26,8 @@ Compatible with OpenSearch 1.1.0
 * Fix bug in paginating multiple bucket paths for Bucket-Level Monitor ([#163](https://github.com/opensearch-project/alerting/pull/163))
 * Various bug fixes for Bucket-Level Alerting ([#164](https://github.com/opensearch-project/alerting/pull/164))
 * Return only monitors for /monitors/_search ([#162](https://github.com/opensearch-project/alerting/pull/162))
+* Add valid search filters ([#191](https://github.com/opensearch-project/alerting/pull/191))
+* Fixed a bug that was preventing the AcknowledgeAlerts API from acknowledging more than 10 alerts at once ([#205](https://github.com/opensearch-project/alerting/pull/205/))
```

Comment:
You might actually want to hold off on updating any `1.1` release notes here.

I think since we're adding patches to the `1.1` branch after the OS 1.1 release, we'll want to update the `1.1` version from `1.1.0.0` to `1.1.1.0` to reflect the patches. In the future if there is a OS 1.1.1 release, then our patched version of Alerting 1.1 would go in there. So it'd actually be going into another release note I guess since we keep them down to the build version (these things wouldn't be done in `main`, we'd do them on the `1.1` branch itself).

This does bring up a question though on our process here. We should increment the patch version if we end up pushing changes to a minor version branch after an OpenSearch release but do we then create release notes whenever we update our patch version or only if an updated version of the plugin is going into an OpenSearch release?

@skkosuri-amzn @lezzago Do you guys agree with the process above and any comments regarding the version update/release notes?

Reply:
That's a good point. I can definitely remove the notes or move them to a new note depending on what's decided. To confirm, we're referring to the 191 and 205 entries, correct?

Reply:
Removed those 2 entries for now while we discuss adding a separate note.